appconfig/appconfig.go

```diff
@@ -92,6 +119,26 @@ func Init() error {
 	return nil
 }
 
+func InitQueryWriter(globalLogsWriter io.Writer) (io.Writer, error) {
+	var queryLogsWriter io.Writer
+	if viper.IsSet("sql_debug_log.path") {
+		if viper.GetString("sql_debug_log.path") != "global" {
+			queryLoggerConfig := logging.Config{
+				LoggerName:  "query",
```

Comment:
In this case, log files will have names like hostname-query-yyyy-mm... 
"query" isn't clear enough in this case. It would be better to use sql-debug or something like that.

Reply:
Fixed